src/status_im/contexts/chat/contacts/events.cljs

```diff
@@ -94,40 +94,56 @@
            [{:method      "wakuext_sendContactRequest"
              :js-response true
              :params      [{:id id :message (or message (i18n/label :t/add-me-to-your-contacts))}]
-             :on-error    [:contact.ui/send-contact-request-failure id]
+             :on-error    [:contacts/send-contact-request-error id]
              :on-success  [:transport/message-sent]}]]]}))
 
 (rf/reg-event-fx :contact.ui/send-contact-request send-contact-request)
 
-(defn send-contact-request-failure
+(defn send-contact-request-error
   [_ [id error]]
   (log/error "Failed to send contact request"
-             {:error error
-              :event :contact.ui/send-contact-request
-              :id    id}))
+             {:id    id
+              :error error
+              :event :contact.ui/send-contact-request}))
```

Comment:
Here's a pattern for the error handlers that has been introduced in other areas. Here we're creating an explicit error handler instead of inlining a function. This way it's more straightforward to unit test this code because it's all data.